src/openms/include/OpenMS/ANALYSIS/OPENSWATH/PeakIntegrator.h

```diff
@@ -48,6 +50,15 @@ namespace OpenMS
     PeakIntegrator();
     virtual ~PeakIntegrator();
 
+    void integratePeak(
+      const MSChromatogram& chromatogram,
```

Comment:
The case of an MSSpectrum could be handled using templates.

Reply:
I'll implement `MSSpectrum` functionality when all three methods will be ready and working for `MSChromatogram`s.